src/main/java/dan200/computercraft/core/apis/http/HTTPRequest.java

```diff
@@ -278,6 +286,11 @@ private static ILuaObject wrapStream( final ILuaObject reader, final int respons
                         // getResponseHeaders
                         return new Object[] { responseHeaders };
                     }
+                    case 2:
+                    {
+                        // getResponseCodeText
+                        return new Object[] { responseCodeText };
```

Comment:
Instead of adding a separate method, I wonder if we could just return both in `getResponseCode`? Might be a slightly cleaner API.

Reply:
Maybe, can I get Lupus's opinion first though?